pom.xml

```diff
@@ -10,14 +10,43 @@
   <parent>
     <groupId>io.burt</groupId>
     <artifactId>jmespath</artifactId>
-    <version>0.2.1</version>
+    <version>0.3.0-SNAPSHOT</version>
```

Comment:
Strictly speaking this should not be changed in a PR, we don't know what the value will be on master when this gets merged (but since this repo is new we can guess, and it's fine)

Reply:
Next time I won't do that. I could not recall it why I changed it. Most probably it was a quick and dirty way to have an updated version (including the new `ArgumentConstraints.listOf` method for variadic parameter list) locally with a different number.